recipe/meta.yaml

```
@@ -9,7 +9,7 @@ source:
   sha256: 723696e93acbe03666213fbc559044f3cae5b8b888b2ddae667402403cff51e5
 
 build:
```

```suggestion
build:
  has_prefix_files:
    Library/lib/pkgconfig/rav1e.pc  # [win]
```
Something like this to replace host prefix in pc file on Windows.